.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+name: Deriv-App
```

Comment:
Can we have a meaningful name?

Reply:
if you see the workflows under this PR
you see it like `Deriv-App / Build And Test (pull_request)`
some how wanted to make more meaningful actually
if i name it build and test it will be Build and test/build and test which is not much useful

Reply:
if you still think that i have to change them let me know, and give me you recommendation too.